repos/system_upgrade/common/actors/setetcreleasever/libraries/setetcreleasever.py

```diff
@@ -0,0 +1,17 @@
+from leapp.libraries.stdlib import api
+
+
+def _set_releasever(releasever):
+    releasever_path = '/etc/dnf/vars/releasever'
+
+    with open(releasever_path, 'w') as fo:
+        fo.write(releasever)
+
+
+def process(facts):
+    target_ver = api.current_actor().configuration.version.target
+
+    # if "/etc/{yum,dnf}/vars/releasever" file exists, let's set it to our
+    # target version.
+    if facts.etc_releasever:
```

Comment:
The comment is not accurate. To make the comment accurate, the code would need to look like this:
```python
    if facts.etc_releasever is not None:
```

currently it does not do the change even when the file exists but it is an empty (which points to a broken configuration) as in such a case the value contains empty string, so nothing happens. From what I understand from our discussion, I guess in such a case we want to set the value as well.